firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -124,19 +133,23 @@
         FirebaseAppDistributionLifecycleNotifier.getInstance());
   }
 
-  /** @return a FirebaseAppDistribution instance */
+  /** Gets the singleton {@link FirebaseAppDistribution} instance. */
   @NonNull
   public static FirebaseAppDistribution getInstance() {
     return FirebaseApp.getInstance().get(FirebaseAppDistribution.class);
   }
 
   /**
    * Updates the app to the newest release, if one is available. Returns the release information or
-   * null if no update is found. Performs the following actions: 1. If tester is not signed in,
-   * presents the tester with a Google sign in UI 2. Checks if a newer release is available. If so,
-   * presents the tester with a confirmation dialog to begin the download. 3. For APKs, downloads
-   * the binary and starts an installation intent. 4. For AABs, directs the tester to the Play app
-   * to complete the download and installation.
+   * null if no update is found. Performs the following actions:
```

Comment:
The first paragraph should only have one sentence, the "summary fragment": http://go/java-practices/javadoc#summary_fragment

The additional content should be their own paragraphs. I think just putting a newline and a `<p>` before "Returns the release information" here works fine.

Reply:
done